Makefile

```diff
@@ -169,7 +169,7 @@ push: image
 .PHONY: test
 test: clean
 	@echo "running unit tests"
-	go test ./... -race -tags deadlock -coverprofile=coverage.txt -covermode=atomic
+	go test ./pkg/... -cover -race -tags deadlock
```

Comment:
We still need ` -coverprofile=coverage.txt -covermode=atomic`. can we add them back?
they are required by the codecov tool to display code coverage.

Reply:
Done